components/camel-tracing/src/main/java/org/apache/camel/tracing/SpanAdapter.java

```diff
@@ -23,24 +23,12 @@ public interface SpanAdapter {
 
     void setError(boolean error);
 
-    void setTag(Tag key, String value);
```

Comment:
These are breaking changes, are we ok with that?

Reply:
See next comment